src/main/java/org/sonarlint/intellij/analysis/DefaultSonarQubeRunnerFacade.java

```diff
@@ -17,42 +17,46 @@
  * License along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
  */
-package org.sonarlint.intellij.inspection;
+package org.sonarlint.intellij.analysis;
 
 import com.intellij.openapi.application.ApplicationInfo;
 import com.intellij.openapi.components.AbstractProjectComponent;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.project.ProjectCoreUtil;
+
 import java.io.File;
 import java.util.Properties;
 import javax.annotation.CheckForNull;
+
 import org.jetbrains.annotations.NotNull;
 import org.sonar.runner.api.EmbeddedRunner;
 import org.sonar.runner.api.IssueListener;
 import org.sonar.runner.api.LogOutput;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.runner.api.LogOutput'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)